benchmark/runner/main.py

```
@@ -818,6 +877,11 @@ def _cmd_run(args: argparse.Namespace) -> int:
         requested_platform = str(args.target_platform or "").strip().lower()
         target_platform = "" if requested_platform == "auto" else requested_platform
     if args.auto_agent_setup:
+        try:
+            _preflight_mobilegym_environment(args.environment_url)
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)